### PR TITLE
Dependencies

### PR DESCRIPTION
**Pull request by goetzk**
This changes includes a number of changes to the instructions around dependencies/prerequisites.

Key changes:
- increase libAV version
- reformat libAV instructions for Windows and Ubuntu
- Remove superfluous instructions from Centos libAV section

### CODE REVIEW

File: guides/InstallationAdminGuide.md [hunk 1]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
 Install the program, taking note of the name and location of the folder in which the avconv and avprobe executable files have been installed, as the openEQUELLA installation will require these details to properly configure and run openEQUELLA.
 
 ### To install Libav for Windows
-1. Go to <http://builds.libav.org/windows/release-gpl/> and download the relevant release build. (e.g. libav-11.3-win64.7z)
+1. Go to <http://builds.libav.org/windows/release-gpl/> and download the relevant release build. (e.g. libav-11.12-win64.7z)
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: While good to use the latest point release, I don't know if folks have specifically tested 11.12.  Unless you know it works, I would not reference it here.

File: guides/InstallationAdminGuide.md [hunk 2]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
...
-4. cd to the directory you want to download libav into.
-5. download the tar.gz file (e.g. <https://libav.org/releases/libav-11.3.tar.gz>)
-6. run 'tar -zxf libav-11.3.tar.gz'
-7. cd into libav-11.3
-8. run './configure --enable-libvo-aacenc --enable-version3 --enable-libx264 --enable-gpl'
-9. run 'make'
-10. run 'sudo make install'
+Install Libav from <https://libav.org/download.html>. libvo_aacenc and libx264 dependencies are also required for video previews to be generated correctly.
+
+1. Install required packages; here we install an assembler,t he audio codec we need and the video codec we encode with.
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: 't he' should be 'the'

File: guides/InstallationAdminGuide.md [hunk 3]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
...
+
+```
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: add 'bash'

File: guides/InstallationAdminGuide.md [hunk 4]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
...
+sudo apt-get install build-essential pkg-config
+sudo apt-get install yasm libvo-aacenc-dev libx264-dev
+```
+
+2. change to the directory you want to download libav into.
+```
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: add 'bash'

File: guides/InstallationAdminGuide.md [hunk 5]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
...
+mkdir scratch
+cd scratch
+```
+
+3. download and extract the tar.gz file, e.g
+```
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: add 'bash'.

Same comment as above about noting libav 11.12

File: guides/InstallationAdminGuide.md [hunk 6]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
...
+wget https://libav.org/releases/libav-11.12.tar.gz
+tar -zxf libav-11.12.tar.gz
+```
+
+4. change into libav-11.12
+```
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: add 'bash'

File: guides/InstallationAdminGuide.md [hunk 7]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
...
+cd libav-11.12
+```
+
+5. configure and make libav; these do NOT need sudo
+```
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: add 'bash'

File: guides/InstallationAdminGuide.md [hunk 8]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
...
+./configure --enable-libvo-aacenc --enable-version3 --enable-libx264 --enable-gpl --prefix=/usr/local
+make
+```
+
+Finally, with sudo, install libav.
+```
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: add 'bash'

File: guides/InstallationAdminGuide.md [hunk 9]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
...
 
 To remove:
-1. cd in the libav-11.3 directory:
-2. run 'sudo make uninstall'
-3. run 'sudo apt-get remove libx264-dev'
-4. run 'sudo apt-get remove libvo-aacenc-dev'
-5. run 'sudo apt-get remove yasm'
-6. delete the libav-11.3 directory and libav-11.3.tar.gz file.
+1. Remove the build dependencies
+```
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: add 'bash'

File: guides/InstallationAdminGuide.md [hunk 10]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
...
+sudo apt-get remove libx264-dev libvo-aacenc-dev yasm
+```
+
+2. Remove libav
+```
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: add 'bash'

File: guides/InstallationAdminGuide.md [hunk 11]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
...
+cd scratch/libav-11.12
+sudo make uninstall
+```
+
+3. delete the libav-11.12 directory and libav-11.12.tar.gz file.
+```
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: add 'bash'

File: guides/InstallationAdminGuide.md [hunk 12]
```diff
@@ -191,7 +197,7 @@ To install Libav:
 * ./configure –enable-static
 * make
 * make install
-* Download libav (For Example: 11.3 - libav-11.3.tar.gz, md5: 1a2eb461b98e0f1d1d6c4d892d51ac9b)
+* Download libav (For Example: 11.12 - libav-11.12.tar.gz, md5: 1a2eb461b98e0f1d1d6c4d892d51ac9b)
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: Same comment on libav 11.12...

File: guides/InstallationAdminGuide.md [hunk 13]
```diff
@@ -117,63 +115,71 @@ Install the program, taking note of the name and location of the folder in which
...
-  * Create a user for openEQUELLA (I did a superuser), and an empty database.
-* sudo yum install ImageMagick
-* Drop an existing 6.4-QA1 install of openEQUELLA into the OS, and confirm openEQUELLA runs.
-* Open your port for tomcat:
-  * firewall-cmd --permanent --add-port=8641/tcp
-  * firewall-cmd --reload
-
-Note: At this point, check your openEQUELLA install that it works and images can be thumbed.
-
-To install Libav:
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md:176]: :+1:  Fair point that this is not needed.

**Review by cbeach47 — CHANGES_REQUESTED** [review]
Besides minor tweaks - I don't know if it's appropriate to reference 11.12 (vs 11.3).  Do you know if 11.12 has been tested with openEQUELLA?

File: guides/InstallationAdminGuide.md (hunk 1 above)
Comment (goetzk) [reply guides/InstallationAdminGuide.md]: I don't know if anyone has tested in production or under a heavy workload. I have only done a basic install for my test VM and haven't loaded any data yet.

**Comment by goetzk** [comment]
@cbeach47 I've made changes WRT all feedback but the lib av version. In the interests of having the branch merged I could add a 'reverting commit' to put them all back to 11.3 (and I imagine open an issue against equella?) if thats what you'd like to see.

**Comment by cbeach47** [comment]
@goetzk - sorry for the delay - yes, please open an issue against the primary code repo for Equella for the community to see if LibAV 11.12 is supported and been confirmed in a production environment.

If you could also rebase this branch, it now has a conflict (or fix the conflict).  I'll make sure there's less lag time reviewing it so we can get it merged.

**Comment by goetzk** [comment]
Raised the question of LibAV version as Equella/676

**Comment by goetzk** [comment]
@cbeach47 reverting changes made - i believe this is now ok to merge.

**Review by cbeach47 — APPROVED** [review]

**Comment by cbeach47** [comment]
:+1:  thanks @goetzk

**Pull request merged**